packages/react-scripts/backpack-addons/splitChunks.js

```diff
@@ -51,6 +51,14 @@ const paths = require('../config/paths');
 const appPackageJson = require(paths.appPackageJson);
 const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
 
+const backpackStylesCacheGroup = {
+  name: 'bpk-styles',
+  type: 'css/mini-extract',
+  chunks: 'all',
+  enforce: true,
+  test: /[\\/]node_modules[\\/]@skyscanner[\\/]backpack-web[\\/]/,
```

Comment:
Why is node_modules in here?

Reply:
We only want to target non-application code, the `@skyscanner/backpack-web` modules will be inside `node_modules`.